ppafm/common.py

```diff
@@ -87,13 +87,13 @@ def Fz2df( F, dz=0.1, k0 = params['kCantilever'], f0=params['f0Cantilever'], n=4
 
 def Fz2df_tilt( F,  d=params['scanTilt'], k0 = params['kCantilever'], f0=params['f0Cantilever'], n=4, units=16.0217656 ):
     '''
-    conversion of vertical force Fz to frequency shift 
+    conversion of vertical force Fz to frequency shift
     according to:
     Giessibl, F. J. A direct method to calculate tip-sample forces from frequency shifts in frequency-modulation atomic force microscopy Appl. Phys. Lett. 78, 123 (2001)
     oscialltion amplitude of cantilever is A = n * dz
     '''
     dr = np.sqrt( d[0]**2 + d[1]**2 + d[2]**2 )
-    hat=np.array(d)/dr
+    np.array(d)/dr
```

Comment:
Once more, does this even do something here.

Reply:
the line does nothing, but the whole function does.
Perhaps I was originaly thingkin I need normalized vectro, but then realized I don't.
Yet I'm not sure about that now. maybe there should be 

return (dFconv_x*hat[0] + dFconv_y*hat[1] + dFconv_z*hat[2] )*units*f0/k0